authz-module/index.md

```diff
@@ -0,0 +1,204 @@
+
+How users decide to use the authz module is up to them. In one case, a validator might want to create a separate account for voting in order to keep their validator key more secure. In another case, a DAO might want to distribute authorizations to members of the DAO in which case a multisg account would grant authorizations to individual accounts and members of the DAO would be able to execute messages without requiring signatures from all the other members.
+
+In this tutorial, you spin up a single node network using the simulation application in Cosmos SDK (`simapp`), grant an authorization to another account, and then execute a message on behalf of the granter as the grantee.
+
+## Requirements
+
+Before you start the tutorial, you need to install the `simd` binary.
+
+Clone the `cosmos-sdk` repository:
+```
+git clone https://github.com/cosmos/cosmos-sdk
+```
+
+Change directories and check out `v0.43.0`:
+```
+cd cosmos-sdk && git checkout v0.43.0
+```
+
+Install the `simd` binary:
+```
+make install
+```
+
+Check to make sure the installation was successful:
+```
+simd version
+```
+
+You should see `0.43.0` printed to the console.
+
+## Configuration
+
+<!-- TODO: update tip to use `unsafe-reset-all` with better user experience -->
+
+::: tip
+If you have used `simd` before, you may already have a `.simapp` directory in you home directory. If you would like to keep the previous data, either save the directory to another location or use the `--home` flag and specify a different directory for each command in the following instructions. If not, remove the previous directory (`rm -rf ~/.simapp`).
+:::
+
+Run the following commands to configure the `simd` binary.
+
+Set the chain ID:
+```
+simd config chain-id demo
+```
+
+Set the keyring backend:
+```
+simd config keyring-backend test
+```
+
+## Key Setup
+
+Create a couple test keys. You will use the `--recover` option so that the addresses used within the example commands below are consistent.
+
+Add a key for Alice:
+```
+simd keys add alice --recover
+```
+
+Enter the following mnemonic:
+```
+plunge hundred health electric victory foil marine elite shiver tonight away verify vacuum giant pencil ocean nest pledge okay endless try spirit special start
+```
+
+Add a key for Bob:
+```
+simd keys add bob --recover
+```
+
+Enter the following mnemonic:
+```
+shuffle oppose diagram wire rubber apart blame entire thought firm carry swim old head police panther lyrics road must silly sting dirt hard organ
+```
+
+## Chain Setup
+
+The following steps will start a chain for the simulation application (`simapp`).
```

Comment:
```suggestion
The following commands start a chain for the simulation application (`simapp`).
```

Reply:
To ensure a common voice, let's use present tense for all of our tutorials, see https://developers.google.com/style/tense 
and https://github.com/cosmos/sdk-tutorials/blob/master/CONTRIBUTING.md 

(let me know what updates we might need to the CONTRIBUTING.md file to help you and others who follow)

Reply:
Thanks! I think the contributing documentation is clear. I will try to make better use of the style guide next time.